include/libtorrent/packet_pool.hpp

```diff
@@ -169,11 +176,11 @@ namespace libtorrent
 			if (allocate <= m_syn_slab.allocate_size) { return m_syn_slab.alloc(); }
 			else if (allocate <= m_mtu_floor_slab.allocate_size) { return m_mtu_floor_slab.alloc(); }
 			else if (allocate <= m_mtu_ceiling_slab.allocate_size) { return m_mtu_ceiling_slab.alloc(); }
-			return packet_ptr(static_cast<packet*>(std::malloc(sizeof(packet) + allocate)));
+			return packet_ptr(packet_mem_manager::create_packet(allocate));
 		}
-		packet_slab<sizeof(utp_header)> m_syn_slab{ 0x100 };
-		packet_slab<(TORRENT_INET_MIN_MTU - TORRENT_IPV4_HEADER - TORRENT_UDP_HEADER)> m_mtu_floor_slab{ 0x200 };
-		packet_slab<(TORRENT_ETHERNET_MTU - TORRENT_IPV4_HEADER - TORRENT_UDP_HEADER)> m_mtu_ceiling_slab{ 0x100 };
+		packet_slab<sizeof(utp_header)> m_syn_slab{ 100 };
+		packet_slab<(TORRENT_INET_MIN_MTU - TORRENT_IPV4_HEADER - TORRENT_UDP_HEADER)> m_mtu_floor_slab{ 512 };
+		packet_slab<(TORRENT_ETHERNET_MTU - TORRENT_IPV4_HEADER - TORRENT_UDP_HEADER)> m_mtu_ceiling_slab{ 256 };
```

Comment:
still, this seems too large. these limits don't determine how many packets you can have in flight, it's how many packets you'll keep around in between bursts. I don't think it's important that these are large numbers. In fact, I think the risk of keeping too much memory tied up for too long is a greater risk